frontend/src/components/AddDelivery.tsx

```diff
@@ -0,0 +1,36 @@
+import {FormEvent, useState} from "react";
+import {NewDeliveryModel} from "../models/DeliveryModel";
+import {Button, TextField, Typography} from "@mui/material";
+import './AddDelivery.css'
+
+type Props = {
```

Comment:
Hier wäre eine etwas deskriptivere Benennung nicht schlecht

Reply:
Changed it to AddDeliveryProps